furms-unity-client/src/test/java/io/imunity/furms/unity/client/users/UserServiceTest.java

```diff
@@ -128,22 +128,22 @@ void shouldReturnEmptyListOfUsers() {
 		assertThat(allUsersFromGroup).isEmpty();
 	}
 
-	@Test
-	void shouldReturnEmptyUsersListFromMultipleGroups() {
-		String path = "/group-members-multi/%2F";
-
-		when(unityClient.post(
-			path,
-			List.of("/fenix/communities/communityId/projects/projectId2/users", "/fenix/communities/communityId/projects/projectId1/users"),
-			Map.of(),
-			new ParameterizedTypeReference<MultiGroupMembers>() {})
-		)
-			.thenReturn(new MultiGroupMembers(List.of(), Map.of()));
-
-		Set<UserPolicyAgreements> allUsersFromGroup = userService.getAllUsersPolicyAcceptanceFromGroups("/", Map.of("communityId", Set.of("projectId1", "projectId2")));
-
-		assertThat(allUsersFromGroup).isEmpty();
-	}
+//	@Test
```

Comment:
@SooLiD this looks incorrect to me, either uncomment the code and fix text or drop it if it does not make sense.

Reply:
I'm merging this to dev, but please address the comment in UserServiceTest.java